x-pack/plugins/observability_solution/exploratory_view/public/components/shared/exploratory_view/rtl_helpers.tsx

```diff
@@ -108,6 +108,7 @@ const defaultCore = coreMock.createStart();
 export const mockCore: () => Partial<CoreStart & ExploratoryViewPublicPluginsStart> = () => {
   const core: Partial<CoreStart & ExploratoryViewPublicPluginsStart> = {
     ...defaultCore,
+    security: undefined,
```

Comment:
`Partial<ExploratoryViewPublicPluginsStart>` has a `security?: securityPluginContract` entry, so `defaultCore.security` wasn't matching the type, which is why I had to override the `security` entry by setting it undefined.